2013-07-13T01:29:59+09:00 ALERT (1): exception 'Zend_Controller_Action_Exception' with message 'Action "aaa" does not exist and was not trapped in __call()' in /Users/Erlkonig/Documents/Workspace/ArenaOfGenelogy/library/Zend/Controller/Action.php:485
Stack trace:
#0 /Users/Erlkonig/Documents/Workspace/ArenaOfGenelogy/library/Zend/Controller/Action.php(518): Zend_Controller_Action->__call('aaaAction', Array)
#1 /Users/Erlkonig/Documents/Workspace/ArenaOfGenelogy/library/Zend/Controller/Dispatcher/Standard.php(295): Zend_Controller_Action->dispatch('aaaAction')
#2 /Users/Erlkonig/Documents/Workspace/ArenaOfGenelogy/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#3 /Users/Erlkonig/Documents/Workspace/ArenaOfGenelogy/index.php(62): Zend_Controller_Front->dispatch()
#4 {main}
